templates/zerver/help/change-who-can-join-a-stream.md

```diff
@@ -1,75 +1,43 @@
 stream browser, labeled **Stream Settings**, will now display the settings of
 the stream that you've selected.
 
     ![Stream selected](/static/images/help/stream-selected.png)
 
-5. After making sure that your cursor is hovering over the **Stream Settings**
-section (right half of the stream browser), scroll down to the **Administrator settings**
-section. This section shows all stream settings that can be modified
-by a realm administrator.
+5. Scroll down to the **Administrator settings** section. This section shows
+all stream settings that can be modified by a realm administrator.
 
     ![Stream administrator settings](/static/images/help/stream-admin-settings.png)
 
-6. To make a public stream private, simply locate the red **Make stream private**
-button located on the bottom of the **Administrator settings** section and click
-on it.
-
-7. After clicking the **Make stream private** button, the button will turn into
-a blue **Make stream public** button, a green section labeled
-**The stream has been made private!** will appear, and the stream's icon will
-switch from a **#** to a lock icon, confirming that the stream has been made private.
+6. To change accessibilty, locate the red **Make stream private** (if your
+stream currently is public) or the blue **Make stream public** (if your stream
+currently is private) button located on the bottom of the **Administrator settings** section.
 
-    ![Public stream private success](/static/images/help/public-stream-private-success.png)
+7. Click the button to change the accessibility, a green pop-up comfirming success
+will be shown.
 
-## Make a private stream public
-
-{!subscriptions.md!}
+    The stream's icon will switch from a **#** to a lock, when the stream is made private.
 
-3. Locate the public stream that you want to make public in the stream browser.
-You can search for specific streams by entering the name of the stream in the
-**Filter streams** search box.
-
-    ![Filter streams box](/static/images/help/filter-stream.png)
-
-4. Click on the stream that you want to make public. The right side of the
-stream browser, labeled **Stream Settings**, will now display the settings of
-the stream that you've selected.
-
-    ![Stream selected](/static/images/help/stream-settings-private.png)
-
-5. After making sure that your cursor is hovering over the **Stream Settings**
-section (right half of the stream browser), scroll down to the **Administrator settings**
-section. This section shows all stream settings that can be modified
-by a realm administrator.
-
-    ![Stream administrator settings](/static/images/help/stream-admin-settings-private.png)
-
-6. To make a public stream private, simply locate the blue **Make stream public**
-button located on the bottom of the **Administrator settings** section and click
-on it.
+    ![Public stream private success](/static/images/help/public-stream-private-success.png)
```

Comment:
You can probably merge lines 39 and 41 to "When a stream is made public, the icon will switch from a lock to a **#**, but when a stream is made private, the icon will switch from a **#** to a lock."